spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkScan.java

```diff
@@ -97,6 +117,36 @@ public static Object[][] parameters() {
     };
   }
 
+  @BeforeAll
```

Comment:
Why is it needed?

Reply:
I don't think this is required, for throwing in new properties you should probably use

```
  protected void withSQLConf(Map<String, String> conf, Action action) {

```

Reply:
Removed and added `withSQLConf`